x/bounty/types/genesis_test.go

```diff
@@ -13,3 +13,108 @@ func TestDefaultGenesisState(t *testing.T) {
 
 	// TODO add GenesisState equal
 }
+
+// func ValidateGenesis(data *GenesisState) error {
+func TestValidateGenesis(t *testing.T) {
+	type args struct {
+		dataGS []GenesisState
+	}
+
+	type errArgs struct {
+		shouldPass bool
+	}
+
+	tests := []struct {
+		name    string
+		args    args
+		errArgs errArgs
+	}{
+		{"genesis(1)  -> success",
+			args{
+				dataGS: []GenesisState{
```

Comment:
I think this can just be a single genesis state?

Reply:
Then we need a test case where we have mutliple genesis states and we compare processing them, but since we don't it's probably better to have it as a single genesis state entry